builds/e2e/e2e.yaml

```
@@ -0,0 +1,87 @@
+# End-to-end tests should run after new docker images and edgelet packages are
+# built, but 'build completion triggers' are not yet supported in YAML. Set to
+# 'none' here and override in the online editor.
+trigger: none
+pr: none
+
+jobs:
+
+################################################################################	
+  - job: linux_arm32v7
+################################################################################	
+    displayName: Linux arm32v7
+
+    pool:
+      name: Azure-IoT-Edge-Core
```

define a variable in the pipeline for re-use and configurable.

```suggestion
      name: $(pool.name)
```

---

Good call, will do.

---

Done